python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_bgp_additional_paths.schema.yml

```diff
@@ -17,8 +17,9 @@ $defs:
           backup: Best path and installed backup path.
           ecmp: All paths in best path ECMP group.
           limit: Limit to n eligible paths.
-          disabled: Disable sending any path.
-        valid_values: [ 'any', 'backup', 'ecmp', 'limit', 'disabled' ]
+          disabled: Disable sending the paths.
+          disabled_any: Disable sending any path.
```

Comment:
Let's remove the disabled_any and revert the change to disabled description.

Reply:
Fixed.